src/python3_pip_skeleton/__main__.py

```diff
@@ -219,6 +221,15 @@ def main(args=None):
     parser = ArgumentParser()
     subparsers = parser.add_subparsers()
     parser.add_argument("--version", action="version", version=__version__)
+    parser.add_argument(
+        "--skeleton-git-url",
```

Comment:
```suggestion
        "--skeleton-org",
```
I would prefer to only set the org to get skeleton from, rather than let it be any url. This means skeleton would need to be forked into that other org with the same repo name